polkadot/node/service/src/lib.rs

```diff
@@ -239,7 +239,7 @@ pub enum Error {
 	InvalidWorkerBinaries { prep_worker_path: PathBuf, exec_worker_path: PathBuf },
 
 	#[cfg(feature = "full-node")]
-	#[error("Worker binaries could not be found, make sure polkadot was built/installed correctly. If you ran with `cargo run`, please run `cargo build` first. Searched given workers path ({given_workers_path:?}), polkadot binary path ({current_exe_path:?}), and lib path (/usr/lib/polkadot), workers names: {workers_names:?}")]
+	#[error("Worker binaries could not be found, make sure polkadot was built and installed correctly. Please see the readme or wiki for the latest instructions. If you ran with `cargo run`, please run `cargo build` first. Searched given workers path ({given_workers_path:?}), polkadot binary path ({current_exe_path:?}), and lib path (/usr/lib/polkadot), workers names: {workers_names:?}")]
```

Comment:
Maybe make a case-decision that describes it for local builds and for downloaded distribution paths?  
Like: if you have the source code locally, run `cargo build`  
if you downloaded it, then also download the two other bins and mark them as executable.  

But yea, linking to some readme maybe good.

Reply:
Do you mean in the error message? Having too much specific info seems like it will be hard to keep up-to-date. Maybe just link to the readme - let me know if that's better.

And did you have to mark them executable? On Mac I just have to accept a pop-up that asks if they are safe to run - should this step be added to the readme?

Reply:
I mean the `chmod +x`, otherwise there is an error that they are not executable. I guess its normal when downloading a binary 🤷‍♂️